src/analysis/normed_space/riesz_lemma.lean

```diff
@@ -0,0 +1,30 @@
+import analysis.normed_space.basic
+import topology.metric_space.hausdorff_distance
+
+variables {k : Type*} [normed_field k]
+variables {α : Type*} [normed_group α] [normed_space k α]
```

Comment:
We switched to meaningful variable names when possible. Here, `E` instead of `α` would be welcome.

Reply:
Made the changes. Also changed `a` and `b` to `x` and `y` to conform better with other files that call their spaces `E` and `F`.